cmd/osctl/pkg/client/client.go

```diff
@@ -49,14 +49,20 @@ type Client struct {
 	NetworkClient networkapi.NetworkServiceClient
 }
 
-// NewClientContextAndCredentialsFromConfig initializes ClientCredentials using default paths
-// to the required CA, certificate, and key.
+// NewClientContextAndCredentialsFromConfig initializes Credentials from config file.
 func NewClientContextAndCredentialsFromConfig(p string, ctx string) (context *config.Context, creds *Credentials, err error) {
 	c, err := config.Open(p)
 	if err != nil {
 		return
 	}
 
+	context, creds, err = NewClientContextAndCredentialsFromParsedConfig(c, ctx)
+
+	return
+}
+
+// NewClientContextAndCredentialsFromParsedConfig initializes Credentials from config file.
```

Comment:
Should this doc string be updated? What makes it different than `NewClientContextAndCredentialsFromConfig`. I can tell by looking at the code, but just pointing out that maybe the doc should be updated?

Reply:
yep, thanks, will do!